src/layer/tile/TileLayer.js

```diff
@@ -35,11 +35,11 @@ L.TileLayer = L.GridLayer.extend({
 	// @aka TileLayer options
 	options: {
 		// @option minZoom: Number = 0
-		// Minimum zoom number.
+		// Minimum zoom number. This will override the [minimum zoom level](#map-minzoom) set on the map.

```

Comment:
Copy-paste from the longer description from `GridLayer`. The reason this is here is because Leafdoc cannot reset the default value of an inherited option while keeping the description.